service/src/main/kotlin/fi/espoo/evaka/assistanceneed/vouchercoefficient/AssistanceNeedVoucherCoefficientQueries.kt

```diff
@@ -63,9 +69,11 @@ fun Database.Transaction.updateAssistanceNeedVoucherCoefficient(
                 """
 UPDATE assistance_need_voucher_coefficient
 SET coefficient = ${bind(data.coefficient)},
-    validity_period = ${bind(data.validityPeriod)}
+    validity_period = ${bind(data.validityPeriod)},
+    modified_at = ${bind(now)},
+    modified_by = ${bind(user.evakaUserId)}
 WHERE id = ${bind(id)}
-RETURNING id, child_id, coefficient, validity_period
+RETURNING id, child_id, coefficient, validity_period, modified_at, (SELECT name FROM evaka_user WHERE id = modified_by) AS modified_by
```

Comment:
Here you are binding the `evaka_user.name` into `AssistanceNeedVoucherCoefficient.modified_by` while in e.g. `insertAssistanceNeedVoucherCoefficient` and `getAssistanceNeedVoucherCoefficientsForChild` you are binding the `user_id` into the same field.